### PR TITLE
Added unique pokemon cries as notification sounds

### PR DESCRIPTION
**Pull request by slvtrs**
## Description
Each mp3 is the original Red/Blue battle cry. The files are titled with the corresponding pokémon's id #, which is appended to the source url within setupPokemonMarker()

## Motivation and Context
Finding a new pokémon wasn't nostalgic _enough_!

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Idhanta** [comment]
Not working, using Safari. I still just get the ding.

**Comment by slvtrs** [comment]
@Idhanta should be fixed now! Thanks for pointing that out.

**Comment by AHAAAAAAA** [comment]
license?

**Comment by brycedorn** [comment]
dang, I just made a PR for this. beat me to it! 👏 

@AHAAAAAAA what's the license like for the icons for each pokemon? seems like the same scenario.

File: static/map.js [hunk 1]
```diff
@@ -315,10 +315,10 @@ function setupPokemonMarker(item) {
 
     if (notifiedPokemon.indexOf(item.pokemon_id) > -1) {
         if (localStorage.playSound === 'true') {
-          audio.play();
+          new Audio(audio_base_url + item.pokemon_id + '.mp3').play();
         }
 
-        sendNotification('A wild ' + item.pokemon_name + ' appeared!', 'Click to load map', 'static/icons/' + item.pokemon_id + '.png', item.latitude, item.longitude);
+        sendNotification('A wild ' + item.pokemon_name + ' appeared!', 'Click to load map', item.pokemon_id, item.latitude, item.longitude);
```
Comment (brycedorn) [inline static/map.js]: could also extract `pokemon_name` here & send it through to `sendNotification` to act more like `ID`, up to you

Comment (slvtrs) [inline static/map.js]: @brycedorn nice, cleaned up the arguments in the following commit accordingly (added pokemon_name and removed the "body: text" string)

**Comment by JZ6** [comment]
@brycedorn we are looking for free to use icon maps rn,
and if you take all responsibility for the usage of these sounds I will merge right now

**Comment by brycedorn** [comment]
@JZ6 I didn't create them so I can't claim ownership or responsibility. but given the [terms of use infractions](http://www.pokemon.com/us/terms-of-use/) that this repo has maintained, I can't imagine this would be the nail in the coffin.

**Comment by AHAAAAAAA** [comment]
Violating a ToS is different than violating copyrights.

**Pull request closed without merging**